packages/superset-ui-legacy-plugin-chart-horizon/src/HorizonChart.jsx

```diff
@@ -78,6 +78,9 @@ class HorizonChart extends React.PureComponent {
       yDomain = d3Extent(allValues, d => d.y);
     }
 
+    if (data.length <= 1)
+      throw new Error('Please select a "Group by" option to enable multiple rows.');
```

Comment:
Will it be possible to have one row even if `group by` is specified? And one record or no result is actually allowed?

![image](https://user-images.githubusercontent.com/335541/77453513-96d05d00-6db4-11ea-8d3a-872c35567b58.png)

(To be honest, I don't know what this chart is for).

Reply:
Agree with @ktmud on supporting one row.

> (To be honest, I don't know what this chart is for).

It is a technique for small multiples of time-series published in 2009, useful when each series has very different max value (e.g one has y = 0-10, another has y = 0-10000. If plot basic line chart with same max y (10000), the first time-series will look like a flat line. If each chart use its own max then 10 on the first chart will look as tall as 10000 on the second chart). Horizon chart was developed to address this situation.

Although less commonly used, since general audience does not know how to interpret it without training. 

https://observablehq.com/@d3/horizon-chart

Original paper:
http://vis.berkeley.edu/papers/horizon/2009-TimeSeries-CHI.pdf

Reply:
Thanks for the explanation, @kristw ! I wonder if there's an opportunity to add a "Pro tip" section to the chart explorer where we display such explanation and tips for best practices (my example query above obviously doesn't make sense).